Sources/TokamakDOM/Views/HTML.swift

```diff
@@ -24,6 +24,7 @@ public typealias View = TokamakCore.View
 public typealias AnyView = TokamakCore.AnyView
 public typealias EmptyView = TokamakCore.EmptyView
 public typealias State = TokamakCore.State
+public typealias Binding = TokamakCore.Binding
 public typealias ObservableObject = TokamakCore.ObservableObject
 public typealias Published = TokamakCore.Published
 public typealias ObservedObject = TokamakCore.ObservedObject
```

Comment:
A little bit off-topic for this PR but it might make sense to re-export these things in a special file. Maybe `TokamakDOM/CoreExports.swift`? I can file an issue/make a PR if it makes sense.

Reply:
Yes, it does make sense, issues/PRs appreciated 👍